pom.xml

```diff
@@ -17,7 +17,7 @@
   <properties>
     <revision>0.7.19</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.60.1</jenkins.version>
+    <jenkins.version>2.76</jenkins.version>
```

Comment:
Nope please use a LTS version.
I would move to 2.107.3 or 2.121.3 looking at the current install base of users who have updated their plugin.